molecule/default/converge.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+  vars:
```

Comment:
Do we actually need these 3 vars here? The role's own defaults already specify these. 

Speaking of which, 

Roll the version and the checksum in the defaults to that of 2.4.2 - version 2.4.2 failed to compile in my local docker test but 2.4.3 worked OK

Reply:
Done.